### ISSUE TITLE
Convert csv segmentation labels to Tensorflow

### ISSUE THREAD

**Issue by achraf-boussaada** [Labelbox/Labelbox#39]
Hi,

- I hope I can make this as clear as possible. If any other details are needed I'll try to provide them.

- I want to use my labeled dataset with the Tensorflow Library. Since I'm using the polygon shape to label the images , some will need for example 5 points to draw a polygon around them and others 6 or 7 !! and when I export the csv file and check it out I'll see that the number of coordinates varies ! which is of course logical. 

- I noticed in the preview section when I want to double check my labeled images that the polygon that I drew will be shown as a box with only 4 coordinates (for all the images). And that exactly what I want.

- In my final csv file I want only 4 coordinates for all the images. Is there anyway that I can label my images with polygons and as I saw in the preview they will turn into boxes with only 4 coordinates and when I export the csv file I get for all the images those coordinates ?

- I know I can simply use the rectangle shape from the beginning but the polygon is more flexible and accurate.

- Well in my opinion I think it's something to add to your project to make it easier for YOLO and Tensorflow users :D ! I hope you take this into consideration

- One last point : when using the rectangle shape to label the images the result will be like this : 
                 `[{x:492, y:1207},{x:492, y:3778},{x:2546, y:3778},{x:2546, y:1207}]`
it would be more helpful if they're generated like so (also to make it more adaptable to YOLO and Tensorflow) : 
                 `[{xmax:2546, xmin: 492},{ymax:3778, ymin:1207}]`

Keep up the good work.

**Comment by manuaero** [comment]
Hi @achraf-boussaada, Thanks for the feedback. The polygons shouldn't change into rectangle at all in the review. Can you share an example URL of the label (from activity tab) where such is the case? We will consider adding that format -- we have plans to add better integration into these frameworks really soon.

**Comment by achraf-boussaada** [comment]
@manuaero this is a link for an example : 
https://app.labelbox.io/projects/cjf88jwk96ihy01018iki80pw/review/cjf9ayu0pljj60186xiq7i7rk

I only selected the white document with a polygon

**Comment by manuaero** [comment]
Hi @achraf-boussaada, could you share the steps to reproduce? It looks like in our records that these annotations were originally created with a rectangle tool.

**Comment by achraf-boussaada** [comment]
Hi @manuaero here are the steps that i did : 
- i uploaded my dataset
- choose image segmentation as a labeling interface
- i configured it as follows : created two labels both are the same but have different tools rectangle and polygon (i think this is the problem)
- i labeled my images

-- when i export the the labels as csv file everything looks normal ! the ones that i labeled as polygons have their correct coordinates [more than 4] which means it's only a glitch.
-- i created a new project and did the same steps and now in the preview i see the correct shapes but not the correct color ! (i choose blue for polygons and green for rectangle ! and it shows me all blue even for rectangles).
-- While doing this i thought of something that may add : 
  - in the activity section under dataset, you can add a miniature of the image in question along with the dataset name and the shape that has been used. Because when you have a dataset of 500 images or more and you want to check something specific it's going to be hard to find it.

- Well I think it's a glitch after all , I can't say if it's from me or it can be fixed from your side.

**Comment by manuaero** [comment]
@achraf-boussaada 
We've made it simpler for you to export your training data from Labelbox into Tensorflow. Now you can export your labeled images in Tensorflow's binary format (.tfrecord). This feature is currently in beta and we'd love to hear how we can further improve this to save you more time.
